mlir/docs/Tutorials/Toy/Ch-4.md

```diff
@@ -91,7 +91,7 @@ struct ToyInlinerInterface : public DialectInlinerInterface {
   /// previously returned by the call operation with the operands of the
   /// return.
   void handleTerminator(Operation *op,
-                        ArrayRef<Value> valuesToRepl) const final {
+                        MutableArrayRef<Value> valuesToRepl) const final {
```

Comment:
Should we just use `ValueRange` everywhere instead of `(Mubtale)ArrayRef<Value>`? The former seems to always return `Value` by value and avoid const-reference annoyance?

Reply:
Done